src/mpcCoreKit.ts

```diff
@@ -196,8 +196,14 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     return this.options.uxMode === UX_MODE.REDIRECT;
   }
 
+  // use dkg is always false for ed25519 key type and for secp256k1 key type when options.useDKG is false
   private get useDKG(): boolean {
-    return this.keyType === KeyType.ed25519 && this.options.useDKG === undefined ? false : this.options.useDKG;
+    return this.keyType === KeyType.ed25519 ? false : this.options.useDKG;
+  }
+
+  // import flow is used when this.options.useDKG is undefined or false
+  private get useImportFlow(): boolean {
+    return !this.options.useDKG;
```

Comment:
i think it might be a bit confusing to have two different getters for the same thing (just inverted). why not just have one and remove the other one?